packages/compass-import-export/src/import/import-json.spec.ts

```diff
@@ -132,7 +132,7 @@ describe('importJSON', function () {
           hasUnboundArray: false,
         });
 
-        const docs = await dataService.find(ns, {});
+        const docs = await dataService.find(ns, {}, { promoteValues: false });
```

Comment:
Had to add this so I could assert against the values without the values being promoted.